src/helpers/client.ts

```diff
@@ -77,9 +77,4 @@ class Client {
   };
 }
 
-export const client = new Client();
-
-export default {
-  client,
-  handlePromiseLikeErrors,
-};
+export default Client;
```

Comment:
I think it's OK to still export `Client` singleton still. 

```suggestion
export const client = new Client();

export default {
  client,
  handlePromiseLikeErrors,
};
```

Reply:
Done!